src/lua/functions/core/game/game_functions.cpp

```diff
@@ -191,6 +191,13 @@ int GameFunctions::luaGameloadMapChunk(lua_State* L) {
 	return 0;
 }
 
+int GameFunctions::luaGameGetExperienceForLevel(lua_State* L) {
+	// Game.getExperienceForLevel(level)
+	const uint32_t level = getNumber<uint32_t>(L, 1);
+	lua_pushnumber(L, level > 0 ? Player::getExpForLevel(level) : 0);
```

Comment:
I think this can be improved, when it is 0 a specific error must be sent so it doesn't seem like everything was ok, since lua and cpp communication are complex, it is advisable to return an error to make it clearer.

Look this: https://github.com/opentibiabr/canary/pull/2636/files

Reply:
I'm not so sure, but I think he's fine now.